docs/BUILD.md

```diff
@@ -1,3 +1,7 @@
+
+
+<p>English | <a href="https://github.com/openvino_tensorflow/docs/I05038-3-BUILD_cn.md">简体中文</a></p>
```

Comment:
Change the file name in the link after changing it. please do it for the links mentioned in all the other files too.